tests/legacy-cli/e2e/utils/project.ts

```diff
@@ -48,7 +48,7 @@ export async function createProject(name: string, ...args: string[]) {
 }
 
 export async function prepareProjectForE2e(name) {
```

Comment:
The `useCIChrome` function also needs to be altered to find the protractor config.  I think it works right now because it's using the system Chrome instead.

Reply:
Shouldn't this already be covered by:
https://github.com/angular/angular-cli/blob/9f985eefac84fffa6d942efa1b1cf8c64c1a4790/tests/legacy-cli/e2e/utils/project.ts#L57

and 

https://github.com/angular/angular-cli/blob/9f985eefac84fffa6d942efa1b1cf8c64c1a4790/tests/legacy-cli/e2e/utils/project.ts#L226

Reply:
According to the PR message, the config will be at `src/e2e/protractor.conf.js`. But the projectDir itself is at `src/` because there is no separate e2e project. So `src/protractor.conf.js` does not exist. Am I missing something here?

Reply:
@filipesilva according to the PR message the `protractor.conf.js` is located in `e2e/protractor.conf.js`

```
├───e2e
│   │   protractor.conf.js
│   │   tsconfig.e2e.json
│   │
│   └───src
│           app.e2e-spec.ts
│           app.po.ts
│
└───src
    │   favicon.ico
    │   index.html
    │   main.po.ts
    │   main.ts
    │   polyfills.ts
    │   styles.css
    │   test.ts
```

Reply:
Discussed offline, I was misunderstanding the changes.